### ISSUE TITLE
Resize, when 200% Zoomed "Home Heading" is not adopting Zoom in "DotNet Home screen": A11y_.NET Core_Template App Home page_Hello World_Resize. 

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/maui#23154]
### GIT Hub Tags:
#AI Limited; #A11yTCS; #A11ySev3; #.NET Core; #E2E-.NET Core-IOS-MAUI-JUNE2024; #Voiceover; #IOS; #WCAG1.4.4; #Resize Text; #FTP; #A11yMAS;

### Environment and OS details:
IOS Version 17.5.1

### Reproduction Steps
Install and open ".NET MAUI Template" app.
.NET MAUI template screen will open.
Observe that Resize, when 200% Zoomed "Home Heading" is not adopting Zoom in "DotNet Home screen".

### Actual behavior:
Resize, when 200% Zoomed "Home Heading" is not adopting Zoom in "DotNet Home screen".

### Expected behavior:
Resize, when 200% Zoomed "Home Heading" should be adopting Zoom in "DotNet Home screen".

### User Impact:
Visually impaired users will face difficulty if Resize, when 200% Zoomed "Home Heading" is not adopting Zoom in "DotNet Home screen".

### Version with bug

9.0.0-preview.4.10690

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

9.0.0-preview.4.10690

### Affected platforms

iOS

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;
